pkg/core/blockchain.go

```diff
@@ -519,6 +521,10 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		// update monitoring metrics.
+		updatePersistedHeightMetric(bHeight)
+		updateHeaderHeightMetric(int(storedHeaderHeight))
```

Comment:
Why not passing `uint32` there? I think there was no conversion before.

Reply:
there are 2 places where it's used.
In one place ( where length - 1) we have int;
This one has uint32

Thought it's a good choice

Reply:
Or actually just drop this line!

Reply:
I see now that `updateHeaderHeightMetric()` is really for `AddHeaders()` and it shouldn't be updated here, no call --- no conversion problem.